src/tools/cephfs/top/cephfs-top

```diff
@@ -279,7 +283,7 @@ class FSTop(object):
                 self.refresh_main_window(x_coord_map, stats_json)
             self.header.refresh()
             self.mainw.refresh()
-            time.sleep(1)
+            time.sleep(self.refresh_interval_secs)
```

Comment:
One side effect of this is that user interruption is checked after the timeout. We probably want to wait using `Condition.wait_for` and call `Condition.notify` in handle_signal().

- https://docs.python.org/3/library/threading.html#threading.Condition.wait_for
- https://docs.python.org/3/library/threading.html#threading.Condition.notify

This can be done in a separate PR.

Reply:
Sure, will do in separate PR